sentry_flink/sentry_flink/flink/flink_translator.py

```diff
@@ -38,17 +39,6 @@ def is_standard_type(type: Any) -> bool:
     return type in FLINK_TYPE_MAP
```

Comment:
This hardcoded type resolution by looking types up in a Map is also kind of haunting me, but will need to tackle in a different PR